### ISSUE TITLE
Inference result was a total black map 

### ISSUE THREAD

**Issue by ztianlin** [weekenddeeplearning/TrackNet#1]
Really appreciated for this excellent work. I have tried to use a lighter backbone to train on my own dataset. The loss converged quickly, however, after the first epoch the output map was totally in black (all pixel values were 0). I thought there was an imbalance between negative samples (black pixels in the heatmap annotation)  and positives. Can you pull me out of this situation? Many thanks.

**Comment by weekenddeeplearning** [comment]
@ztianlin - Sorry about getting back late. I had put together this repo by contacting the authors and using their repo below. We would have to go through their train module to understand what's going on. Did you verify the annotation heatmaps and label csv before the training?

[https://nol.cs.nctu.edu.tw:234/open-source/TrackNet/tree/master]

**Comment by ztianlin** [comment]
Well, It seems that I implemented the network badly. In your keras implementation, the normalization method is layer normalization instead of batch normalization, right?

**Comment by masouduut94** [comment]
So any luck with the training?
Every time I run training, I get errors during epochs 1-7.
@ztianlin
